### PR TITLE
Added support for linting with Vivado Simulator's xvlog.

### PR DESCRIPTION
**Pull request by HansLehnert**
I added support for xvlog which is distributed along with Xilinx's Vivado tool.

Also restructured the extension for easier addition of linters.

### CODE REVIEW

**Comment by mshr-h** [comment]
Thanks for your contribution!
I'll review it until this weekend.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "VerilogHDL",
```
Comment (mshr-h) [inline package.json]: Any reason to change the extension name?

Comment (HansLehnert) [reply package.json]: The "name" field should be lowercase pony as described in https://code.visualstudio.com/docs/extensionAPI/extension-manifest

The old name should be moved to "displayName"

Comment (mshr-h) [reply package.json]: Ok.

**Comment by mshr-h** [comment]
Which platform have you tested on? (Windows?, Ubuntu?, macOS?)

**Comment by HansLehnert** [comment]
Only on Windows. I will eventually test it in Ubuntu.

**Comment by mshr-h** [comment]
Ok.

**Pull request merged**